.travis.yml

```diff
@@ -204,54 +204,3 @@ before_cache:
 notifications:
   slack:
     secure: NPHH9nIuInY3v4Jvl1ywC5BlygY9YiVni6LLcR52nwdx/FocO9j80aNKx/kuso+nGRM3dmWKPWdk8ShOOS2ZRvYXQUe2Gy6wSnlwqjKGaHEReZA5hOFuYfTQw0Gh7WGzwiuYpvAzhXzn31GvL92M24Oq0kqP64sA9nndBAT3uMv5PdQV40JtL8kQdh66wHuwtJ/LrljWKTwrTJH8NcGc7I6MLljxe+3n1zy49J7MqVyGdBdFA6CMc6Abgc3XKfsadJH3QFC05LfyOQqMjCutSd+HWkl7yMFLWpddLj0C9DdpG1z7324gcQjlLbDGI3Mc3vd/bTLnanIgQ4ZIWSK8/0j8u/dJw0DPUkO82sZ+1nvtwNvE+C67M2vxlunRuxEwEIw7a6cUuguKh/EhZYDccF6523kFPyCLs8zSKdScCjjz4pj+jj3niug/tBiSRebwhcfLnWlQp7trY54ETtvwjAidGfXwuqO07FsNJL2gweg+axFhcSqFos7VwWeJVB1bVGg1PYM3wFarr25aQstIvE+Ap9Mym1MXXhswwe09BYQiOzoiK/CYAKrQT3IjLBGOM8P+uh2rv4w+wUNlgfUnG9A9aiRAlw+9uXDgFIR59y3SylKtWabDtX5khO/5TTlzc4Cq3tGSoNPPJ+WAtUISGrgCjQAVVnhdZ7chi4aLt5s=
-
-before_deploy:
-  - mkdir /tmp/artifacts
-  - cp -n presto-server/target/presto-server-*.tar.gz /tmp/artifacts
-  - cp -n presto-server-rpm/target/presto-server-rpm-*.x86_64.rpm /tmp/artifacts
-  - cp -n presto-product-tests/target/presto-product-tests-*-executable.jar /tmp/artifacts
-  - cp -n presto-jdbc/target/presto-jdbc-*.jar /tmp/artifacts
-  - cp -n presto-cli/target/presto-cli-*-executable.jar /tmp/artifacts
-  - echo $TRAVIS_COMMIT > /tmp/artifacts/git-revision.txt
-  - echo "<script>location='https://travis-ci.org/${TRAVIS_REPO_SLUG}/builds/${TRAVIS_BUILD_ID}'</script>"
-    > /tmp/artifacts/travis_build.html
-  - ls -lah /tmp/artifacts
-
-deploy:
-  on:
-    all_branches: true
-    condition: -v DEPLOY_S3_ACCESS_KEY && -v MAVEN_CHECKS
-  provider: s3
-  access_key_id: ${DEPLOY_S3_ACCESS_KEY}
-  secret_access_key: ${DEPLOY_S3_SECRET_KEY}
-  bucket: ${DEPLOY_S3_BUCKET}
-  skip_cleanup: true
-  local-dir: /tmp/artifacts
-  upload-dir: ${ARTIFACTS_UPLOAD_PATH}
-  acl: public_read
-
-after_script:
-- |
-  if [[ -v DEPLOY_S3_ACCESS_KEY ]]; then
-    sudo pip install awscli
-    export AWS_ACCESS_KEY_ID="${DEPLOY_S3_ACCESS_KEY}"
-    export AWS_SECRET_ACCESS_KEY="${DEPLOY_S3_SECRET_KEY}"
-
-    JOB_ARTIFACTS_URL_PREFIX="s3://${DEPLOY_S3_BUCKET}/${ARTIFACTS_UPLOAD_PATH}/travis_jobs/${TRAVIS_JOB_NUMBER}-run"
-    JOB_RUN_ATTEMPTS=$( aws s3 ls "${JOB_ARTIFACTS_URL_PREFIX}" | wc -l | tr -d '[:space:]' )
-    JOB_STATUS=$( [ "${TRAVIS_TEST_RESULT}" == "0" ] && echo SUCCESS || echo FAILURE )
-
-    mkdir -p /tmp/job_artifacts/
-    rsync -av -m \
-      --include='**/' \
-      --include='**/surefire-reports/**.xml' \
-      --include='**/surefire-reports/emailable-report.html' \
-      --include='**/product-tests-presto-jvm-error-file.log' \
-      --include='**/test-reports/junitreports/**.xml' \
-      --include='**/test-reports/emailable-report.html' \
-      --exclude='*' \
-      . /tmp/job_artifacts/
-    wget "https://api.travis-ci.org/jobs/${TRAVIS_JOB_ID}/log.txt?deansi=true" -O /tmp/job_artifacts/log.txt
-
-    aws s3 sync /tmp/job_artifacts "${JOB_ARTIFACTS_URL_PREFIX}_$((JOB_RUN_ATTEMPTS + 1))-${JOB_STATUS}" --quiet
```

Comment:
- typo in commit
- i think we actually need this. When jobs are failing with eg OOM, we get no hs dump and one would be useful.